docs/rules/switch-final-break.md

```diff
@@ -0,0 +1,32 @@
+# Checks whether the final clause of a switch statement ends in unnecessary `break;`. (switch-final-break)
+
+Examples of **correct** code:
+
+```js
+switch (x) {
+  case 0: {
+    foo();
+  }
+}
+```
+
+Examples of **incorrect** code:
+
+```js
+switch (x) {
+  case 0: {
+    foo();
+    break;
+  }
+}
+```
+
+## Options
+
+Can get `never` (the default) or `always` (always ends with `break;`):
```

Comment:
I think we can improve the clarity of this line by formatting it similarly to this section if the  [`rest-spread-spacing` documentation](https://eslint.org/docs/rules/rest-spread-spacing#options).